File: dynamicpdf-vbnet-examples/Examples/CreatePDF.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.PageElements

Namespace DynamicPDFCoreSuite.Examples
    Class CreatePDF
        Public Shared Sub Run()
            Dim document As New Document()
            Dim outputPath As String = Util.GetPath("Output/CreatePDF-output.pdf")

            Dim page As New Page(PageSize.Letter, PageOrientation.Portrait, 54.0F)
            document.Pages.Add(page)

            Dim labelText As String = "Hello World..." & vbLf & "From DynamicPDF Core Suite for .NET" & vbLf & "DynamicPDF.com"
            Dim label As New Label(labelText, 0, 0, 504, 100, Font.Helvetica, 18, TextAlign.Center)
            page.Elements.Add(label)

            document.Draw(outputPath)
        End Sub
    End Class
End Namespace

File: dynamicpdf-csharp-examples/Examples/CreatePDF.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.PageElements;

namespace DynamicPDFCoreSuite.Examples
{
    class CreatePDF
    {
        public static void Run()
        {
            Document document = new Document();

            Page page = new Page(PageSize.Letter, PageOrientation.Portrait, 54.0f);
            document.Pages.Add(page);
            
            string labelText = "Hello World...\nFrom DynamicPDF Core Suite for .NET\nDynamicPDF.com";
            Label label = new Label(labelText, 0, 0, 504, 100, Font.Helvetica, 18, TextAlign.Center);
            page.Elements.Add(label);
            
            document.Draw(Util.GetPath("Output/CreatePDF.pdf"));
        }
    }
}
